<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_source">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>371</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>source directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_target">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>401</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>target directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_chg_source_dir">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Change source directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_chg_target_dir">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Change target directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_crt_copy_list">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Copy List</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPLs">
    <property name="title">
     <string>PLs</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuPLs"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
